DNN Platform/Library/Entities/Tabs/TabVersions/TabVersionController.cs

```diff
@@ -112,6 +119,34 @@ private static string GetTabVersionsCacheKey(int tabId)
         {
             return string.Format(DataCache.TabVersionsCacheKey, tabId);
         }
+
+        private IEnumerable<TabVersion> ApplyUserLocalTime(IEnumerable<TabVersion> tabVersions, TimeSpan userUtcOffset, TimeZoneInfo userTimeZone)
+        {
+            return tabVersions.Select((tabVersion) =>
+            {
+                var serverUtcOffset = TimeZoneInfo.Local.BaseUtcOffset;
+                if (serverUtcOffset.CompareTo(userUtcOffset) >= 0)
+                {
+                    tabVersion.CreateOnUserLocalDate = ApplyDaylightSavingHour(userTimeZone, tabVersion.CreatedOnDate.Subtract(serverUtcOffset).Add(userUtcOffset));
+                }
+                else
+                {
+                    tabVersion.CreateOnUserLocalDate = ApplyDaylightSavingHour(userTimeZone, tabVersion.CreatedOnDate.Add(serverUtcOffset).Subtract(userUtcOffset));
+                }
+                return tabVersion;
+            });
+        }
+
+        private DateTime ApplyDaylightSavingHour(TimeZoneInfo info, DateTime localTime)
+        {
+            if(info.IsDaylightSavingTime(localTime))
+            {
+                return localTime.AddHours(1);
```

Comment:
I'm pretty sure this is incorrect. `TimeZoneInfo` is perfectly aware of daylight saving in specific time zones